bazel/external/googleurl.patch

```diff
@@ -92,13 +92,29 @@ index 6651220..a469c19 100644
  #else
  #define REINITIALIZES_AFTER_MOVE
  #endif
+diff --git a/base/containers/checked_iterators.h b/base/containers/checked_iterators.h
+index b5fe925..31aa81e 100644
+--- a/base/containers/checked_iterators.h
++++ b/base/containers/checked_iterators.h
+@@ -237,9 +237,11 @@ using CheckedContiguousConstIterator = CheckedContiguousIterator<const T>;
+ // [3] https://wg21.link/pointer.traits.optmem
+ namespace std {
+ 
++#ifdef SUPPORTS_CPP_17_CONTIGUOUS_ITERATOR
```

Comment:
Can you add some comment here in the patch just to indicate what is going on / what the implication of this is? Also, are we tracking this upstream in any way? cc @RenjieTang @dio @yanavlasov